cmd/state-tray/main.go

```diff
@@ -16,32 +17,56 @@ func main() {
 }
 
 func onReady() {
+	err := run()
```

Comment:
So is the ONLY error that can happen at the moment the "About" click? The systray stuff just doesn't throw errors?

Reply:
Correct, nothing in the `systray` library returns an error. I'm not sure if this `run` function is necessary anymore if we are going to be logging the errors when they occur. Do you still want it?

Reply:
Let's leave `run` where it is, that at least establishes the contract we want to keep long term.